src/textalloc/__init__.py

```diff
@@ -5,7 +5,7 @@
 )
 import numpy as np
 import time
-from typing import List, Union
+from typing import Any, Dict, List, Optional, Union
```

Comment:
Remove Optional here and below, not currently used for any other optional arguments.

Reply:
Ok, but it would be better to add it to the others ;)

Reply:
I agree, just didn't want to grow the task for you, will add it if I am in the mood for some refactoring ;)